birdcage_frontend/templates/notifications_settings.html

```diff
@@ -64,15 +64,15 @@ <h3>Notifications Powered by Apprise</h3>
 const jwt_token = localStorage.getItem("jwt_token");
 
 function getNotificationServices() {
-  return fetch(`${API_SERVER_URL}/services`, {
+  return fetch(`${API_SERVER_URL}/api/services`, {
```

Comment:
More standardization, took me a while to figure out why API endpoints weren't working only to realize they didn't all start with the same prefix.